static/src/javascripts-legacy/bootstraps/enhanced/main.js

```diff
@@ -163,10 +163,17 @@ define([
         }
 
         // use a #force-sw hash fragment to force service worker registration for local dev
-        if ((window.location.protocol === 'https:' && config.page.section !== 'identity') || window.location.hash === '#force-sw') {
+        if ((window.location.protocol === 'https:' && config.page.section !== 'identity') || window.location.hash.indexOf('force-sw') > -1) {
```

Comment:
Done. I do not do the whole URL test (the promise will just never resolve if the SW is not there) but I can add it if you prefer